### DISCUSSION TITLE
No way to install under MSYS2

### DISCUSSION THREAD

**Discussion by antoniopaolini** [jrnl-org/jrnl#1413] [General]
Hello everybody,
I tried to install jrnl under my MSYS2 system, but no way to get it.
Everytime I can't obtain all the dependencies satisfied. first zlib headers, next time Pillow,  and so on.
Then, the last attempt, when I reached the best situation manually, pipx error (I hatePython...)

I don't know if it's a jrnl or msys issue.

The best I have reached is tryng to install with:
```
$ python3 -m pip install --user pipx
[...everything OK...]
$ python3 -m pipx ensurepath
Success! Added C:/Users/MYUSER/.local/bin to the PATH environment variable.

Consider adding shell completions for pipx. Run 'pipx completions' for
instructions.

You will need to open a new terminal or re-login for the PATH changes to take
effect.

Otherwise pipx is ready to go!

$ pipx install jrnl
bash: pipx: comando non trovato
[NdR: TRANSLATED: command not found]
```
Repeat, I HATE Python... :-(

I tried to so this:
```bash
$ python3 -m pipx install jrnl
creating virtual environment...
creating shared libraries...
creating shared libraries...
creating shared libraries...
creating shared libraries...
creating shared libraries...
creating shared libraries...
creating shared libraries...
creating shared libraries...
creating shared libraries...
```
Infinite loop...

I tried also to do:
```
$ python3 -m pip install jrnl
```
Buth this lead me to another failure:
![image](https://user-images.githubusercontent.com/60228211/153824929-1d4ad685-e784-4a5f-aca5-5bca979c575e.png)
![image](https://user-images.githubusercontent.com/60228211/153824956-832b3958-3b3b-4eee-9039-790d8217d9cf.png)
I updated pip, manually installed rust compiler and criptography package, 

I give up.
Does anyone have any advice for me?

**Comment by micahellison** [comment]
Hi there, it looks like pipx installed itself correctly, but the path isn't set appropriately. I've run into this on Windows multiple times due to the way it sets paths globally without updating your terminal session's path.

If you start a new terminal, does pipx work? If it does, running `pipx install jrnl` should do the trick. If not, see the Windows section of the pipx documentation for more info on dealing with this pipx path issue.

If that still doesn't work, you might want to consider a different way of installing Python -- I've never used MSYS2, but I can vouch for the installer download from the python.org site, and I have also been able to work it through WSL/Ubuntu using apt to get Python.
